src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -132,20 +132,20 @@
 
         <div class = "container-fluid ${courseConfigurationForm.gradingOn? '' : 'hidden'}" id="conversionConfig" >
             <br/>
-            <p> <strong>NOTE:</strong> When this assignment is pushed to the gradebook, it will immediately be published.
+            <p> <strong>NOTE:</strong> Select “Save Setup” every time changes are made. When this assignment is pushed to the gradebook, it will immediately be published.
                 <strong>Please DO NOT alter the assignment in the gradebook, but instead use this tool to update the assignment as needed.</strong>
                 Click the "Convert Attendance to Assignment" checkbox again to remove the assignment.
             </p>
             <br/>
             <div class="col-md-2 col-md-offset-0">
                 <label for="assignmentName">
                     <h5><i>Assignment Name: </i></h5>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)